### ISSUE TITLE
Tooltip doesn't appear for HTML elements in XUL

### ISSUE THREAD

**Issue by dstillman** [zotero/zotero#1947]
Neither `title` nor `tooltiptext` works.

We'll need this more soon, so I'm working on a workaround for this.

**Comment by adomasven** [comment]
@dstillman So I've just run across this issue working on https://github.com/zotero/zotero/issues/2321 and I think @mrtcode had found an elegant solution to this here: https://github.com/zotero/zotero/blob/d361930bcc4730630483625a2d7cf484f1c27554/chrome/content/zotero/zoteroPane.xul#L248-L249 , which itself was done at least 3 months prior to this. I think it's based on [this stackoverflow thread](https://stackoverflow.com/questions/10684653/tooltips-titles-in-xul-browser-content-are-not-working) and [this bugzilla bug](https://bugzilla.mozilla.org/show_bug.cgi?id=451792#c1)(to some extent).

Basically you add `tooltip="html-tooltip"` to the XUL element above the one which contains the HTML elements and tooltips on "Title" attribute start working. So https://github.com/zotero/zotero/commit/e9e434cd2ecda0c5b0dd8ad6dde1c0db1eec2882 can be drastically simplified/removed.

P.S. I'm going to need to modify Martynas' work a bit there since it's searching for the [title] attribute too broadly.
